errors/error.go

```diff
@@ -6,7 +6,9 @@ var (
 	// ErrNoContent custom error for no record found
 	ErrNoContent = errors.New("no model exist in database")
 	// ErrConflict custom error for record already exist
-	ErrConflict         = errors.New("user model is already exist in database")
+	ErrConflict = errors.New("user model is already exist in database")
```

Comment:
we should use a generalised message here because the model can't always be `user` or we can use `ErrUserConflict` just like you did for other models below.

Reply:
Done.